exporters/dsexporter/README.md

```diff
@@ -0,0 +1,25 @@
+# dsexporter
+
+## Overview
+
+The `dsexporter` is a specialized Prometheus exporter designed to provide availability metrics for the components
+within the Konflux ecosystem. It generates a custom metric (`request_count`) related to the number of requests handled
+by an HTTP handler, enabling the evaluation of overall availability.
+
+## Purpose
+
+In the context of the Konflux ecosystem, the `dsexporter` serves as an Availability Exporter, contributing to the
+evaluation of the availability of each component within the system. By exposing the `request_count` metric, it allows
+for monitoring and analyzing the workload and performance of the associated service.
+
+## Prometheus Exporters
+
+A **Prometheus exporter** exposes metrics from a system or application in a format that Prometheus can ingest and monitor.
+The `dsexporter` follows the Prometheus exposition format, providing a web endpoint that outputs metrics in a well-known format.
+
+## Getting Started
+
+To use the `dsexporter`, follow these steps:
+
+1. Import the necessary packages in your code.
+2. Start the HTTP server and the exporter will be accessible at the respected endpoint.
```

Comment:
While the concept of `dsexporter`s was explained, I feel as this section in lacking details.

Reply:
Do you think this previous Getting Started paragraph is redundant now ? As I see it was completely removed.

Reply:
Keeping it brief and removing details was my thought.

Reply:
Keep it brief is fine, but can this information be found anywhere else? 
Do you think this information is important to someone who will want to use dsexporter?  - if not, then I'm fine with this change.